### DISCUSSION TITLE
theme.park

### DISCUSSION THREAD

**Discussion by lewisakura** [catppuccin/catppuccin#1881] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

https://docs.theme-park.dev/custom/

### Has the requested port been themed already?

https://github.com/lewisakura/theme.park

### Any additional comments?

as previously mentioned in discord, latte is on a best case basis. theme.park is a theme framework for many apps, therefore it is impossible to fix it up on every one it supports (at time of writing, 50).

**Comment by lewisakura** [comment]
tested so far is sonarr, radarr, ombi, jellyfin, qbittorrent web, and vuetorrent with no obvious issues (sans latte). overseerr has a couple of issues with visibility on buttons but might be adjustable.

**Comment by sgoudham** [comment]
Transferred over to #1882
